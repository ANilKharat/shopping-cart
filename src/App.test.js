import React from 'react';
import { mount } from 'enzyme';
import App from './App'
import { Provider } from 'react-redux'
import store from '../src/redux/store/store'


describe('Products page test case', () => {
    it("Snapshot test case", () => {
        const component = mount(
            <Provider store={store}>
                <App />
            </Provider>
        )
        expect(component).toBeTruthy()
    })
})