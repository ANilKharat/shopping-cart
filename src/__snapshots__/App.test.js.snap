// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Products page test case Snapshot test case 1`] = `ReactWrapper {}`;
